api/data_explorer/util/filters_facet.py

```diff
@@ -0,0 +1,31 @@
+"""Subclass of Facet for performing faceted search with bucketed filter expressions."""
+
+from elasticsearch_dsl.faceted_search import Facet
+from elasticsearch_dsl.query import Filtered
+
+
+class FiltersFacet(Facet):
+    """Custom facet for creating aggregation buckets based on filters that can span multiple fields."""
+    agg_type = 'filters'
+
+    def __init__(self, filters):
+        self._filters = filters
+        super(FiltersFacet, self).__init__(filters=filters)
+
+    def get_values(self, data, filter_values):
+        """
+        Slight modification of the base class because the response format for filters aggregations is 
+        slightly different:
+        https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-filters-aggregation.html
```

Comment:
Can you include this link in either the pydoc at the top of this file, or pydoc for this class?